B827EBFFFEE51011.json

```diff
@@ -0,0 +1,19 @@
++{
++      "gateway_conf": {
```

Comment:
Please check the content of the file, it's full of plus signs at the start of each line.